test/Interop/Cxx/reference/reference-silgen.swift

```diff
@@ -36,12 +36,12 @@ func getConstCxxRvalueRef() -> UnsafePointer<CInt> {
 
 func setCxxRef() {
   var val: CInt = 21
-  setStaticIntRef(&val)
+  setStaticIntRef(val)
 }
 
 // CHECK: sil hidden @$s4main9setCxxRefyyF : $@convention(thin) () -> ()
-// CHECK: [[REF:%.*]] = function_ref @{{_Z15setStaticIntRefRi|\?setStaticIntRef@@YAXAEAH@Z}} : $@convention(c) (@inout Int32) -> ()
-// CHECK: apply [[REF]](%{{[0-9]+}}) : $@convention(c) (@inout Int32) -> ()
+// CHECK: [[REF:%.*]] = function_ref @{{_Z15setStaticIntRefRi|\?setStaticIntRef@@YAXAEAH@Z}} : $@convention(c) (Int32) -> ()
```

Comment:
Questions: 
1. If I understand correctly these are executable tests to check the sil? 
2. Why don't I need the `@in` keyword here compared to the tests I updated on lines 61 and 62?